File: Assignments/fizzbuzz.pl
#!/usr/bin/perl

#variable values

my $bear; 

print "enter number between 1 and 100\n";

$bear = <STDIN>;
chomp ( $bear );
 
if(($bear % 3)==0){

print "fizz\n"; }

if(($bear % 5)==0){

print "buzz\n"; }

if (($bear % 3 ==0) && ($bear % 5 ==0)) {

print "fizzbuzz\n"; }

File: Assignments/fizzbuzz.py
#!/usr/bin/python
#Variable values


print("enter a number value:")

num = int(input())
if num % 3 == 0 and num % 5 == 0 :
    print("Fizzbuzz")
elif num % 3 == 0 :
    print("fizz")
elif num % 5 == 0 :
    print("Buzz")
else: 
    print (num)
